zzzz_modular_occulus/code/modules/projectiles/projectile/projectile.dm

```diff
@@ -1,6 +1,9 @@
+/obj/item/projectile/change_knockback(change)
+	knockback = initial(knockback) + change
```

Comment:
Does this decay over time or does knockback continue building over multiple gunshots (not in rapid succession?)

Reply:
I am not sure I understand the question here. If a projectile has knockback and it is changed to 0 then it simply acts as a projectile that has knockback = 0 and if it is changed from 0 to 1 it now gives it knockback.